Config/ConfigLoader.cs

```
@@ -0,0 +1,15 @@
+﻿using System.IO;
+using System.Text.Json;
+
+namespace WeatherBots.Config
+{
+    public static class ConfigLoader
+    {
+        public static BotConfigRoot Load(string path)
+        {
+            var txt = File.ReadAllText(path);
+            var opts = new JsonSerializerOptions { PropertyNameCaseInsensitive = true };
+            return JsonSerializer.Deserialize<BotConfigRoot>(txt, opts) ?? new BotConfigRoot();
```

It would be a better practice here to throw an exception if the serialization couldn't happen rather than silently suppressing the error.